src/components/maze/Gameboard.tsx

```diff
@@ -84,14 +85,20 @@ export function Gameboard({
     // return `${styles.mazeCell} nonPathColorSet${selectedColorSet} ${backgroundColor}${selectedColorSet}`;
     return `${styles.mazeCell} ${backgroundColor}${selectedColorSet}`;
   }
+
   function getPlayerTileClasses(cell: MazeTileData) {
-    let backgroundImage;
-    if (cell.fight || cell.enemyWon || cell.hasCartel || cell.hasExit) {
-      backgroundImage = 'playerBackgroundEmpty';
-    } else {
-      backgroundImage = 'playerBackgroundElementOnTop';
-    }
-    return `${styles.mazeCell} ${styles.playerCell} ${getPlayerImgDirection()} ${backgroundImage}`;
+    const backgroundImage =
+      cell.fight || cell.enemyWon || cell.hasCartel || cell.hasExit
+        ? 'playerBackgroundEmpty'
+        : 'playerBackgroundElementOnTop';
+
+    // Combine multiple CSS classes using the 'clsx' utility function to handle conditional styling
+    return clsx(
```

Comment:
We have multiple lines in the project where we should use this function to be consistent on the way to do this stuff. I know it's not a priority right now to refactor the code but it doesn't look good idea to comment every place where we use this function. I was expecting some better documentation, like creating a local function just to call the clsx but adding a description so it could be seen while hovering the function on the code or something like that

Reply:
we can do that in refactoring session.

Reply:
Yeah but if that can be done in refactoring season, ¿Why did you change this now? ¿Why did you add clsx function in this PR?

It's not a braking thing, the code will be as good as before but let's try to not do PR's with so many tasks because this particular change is not compleate/shouldn't be done now i think.